docker/openziti-mono/Dockerfile

```diff
@@ -15,15 +15,11 @@ RUN apt update && \
     jq \
     clang-9 \
     libclang-9-dev
-RUN wget https://github.com/dotnet/docfx/releases/download/v2.56.5/docfx.zip && \
-    unzip docfx.zip -d docfx && \
-    rm docfx.zip && \
-    echo "mono /docfx/docfx.exe \$@" > /usr/bin/docfx && \
-    chmod +x /usr/bin/docfx && \
-    wget -q -c http://doxygen.nl/files/doxygen-1.8.20.linux.bin.tar.gz -O - | tar -xz && \
-    ln -s /doxygen-1.8.20/bin/doxygen /usr/bin/doxygen && \
-    curl -s https://api.github.com/repos/netfoundry/ziti-ci/releases/latest | jq -r '.assets[] | select (.name=="ziti-ci") | .browser_download_url' | wget -i - -O ziti-ci && \
-    chmod +x ziti-ci
-    #&&
-    #\
-    #rm -rf /var/lib/apt/lists/* /tmp/*
+RUN wget https://github.com/dotnet/docfx/releases/download/v2.56.5/docfx.zip

```

Comment:
Would you mind making this one big RUN command to cut down on layer size? I'm no pro but I think that it'd make a difference. Also I noticed that the docfx version is old. Can you use 2.59.3?